src/exports/Image/index.js

```diff
@@ -194,7 +194,7 @@ const Image: React.AbstractComponent<ImageProps, React.ElementRef<typeof View>>
     const selectedSource = shouldDisplaySource ? source : defaultSource;
     const displayImageUri = resolveAssetUri(selectedSource);
     const imageSizeStyle = resolveAssetDimensions(selectedSource);
-    const backgroundImage = displayImageUri ? `url("${encodeURI(displayImageUri)}")` : null;
+    const backgroundImage = displayImageUri ? `url("${displayImageUri}")` : null;
```

Comment:
Could we throw an exception here if `displayImageUri` contains a double quote? This shouldn't happen now as it's already encoded, but will be a check for future versions to behave properly.

Reply:
Okay added a check for uri includes double quote.